server/app/views/admin/programs/ProgramApplicationTableView.java

```diff
@@ -0,0 +1,538 @@
+package views.admin.programs;
```

Comment:
Non-blocking (no action required): This should be tested in a browser test, but I see there are issues to finish adding tests.